File: app/src/main/java/com/example/muhammadjon/myproject/activities/IWindowView.java
package com.example.muhammadjon.myproject.activities;

import com.example.muhammadjon.myproject.dbase.Merchants;

import java.util.List;

public interface IWindowView {

    void onHidePb();

    void onShowPb();

    void onResult(List<Merchants> merchants);

    void onError(String text);
}

File: app/src/main/java/com/example/muhammadjon/myproject/activities/IWindowView.kt
package com.example.muhammadjon.myproject.activities

import com.example.muhammadjon.myproject.dbase.Merchants

interface IWindowView {

    fun onHidePb()

    fun onShowPb()

    fun onResult(merchants: List<Merchants>)

    fun onError(text: String)
}
